tableau-specialist/1.0/modules/ROOT/pages/tableau-specialist-connector-reference.adoc

```diff
@@ -0,0 +1,1255 @@
+| Config Ref a| ConfigurationProvider |  Name of the configuration used to execute this component. |  | x
+| Streaming Strategy a| * <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configures how Mule processes streams. The default is to use repeatable streams. |  |
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation’s output. The outcome of the expression is stored in the *Target Variable* field. |  #[payload] |
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Binary
+| Attributes Type a| Binary
+|===
+
+=== For Configurations
+
+* <<TableauSpecialistConfig>>
+
+=== Throws
+
+* TABLEAU-SPECIALIST:CONNECTIVITY
+* TABLEAU-SPECIALIST:FORBIDDEN
+* TABLEAU-SPECIALIST:INSUFFICIENT_SITE_STORAGE_REMAINING
+* TABLEAU-SPECIALIST:PROJECT_ID_MISMATCH
+* TABLEAU-SPECIALIST:RETRY_EXHAUSTED
+* TABLEAU-SPECIALIST:TIMEOUT
+* TABLEAU-SPECIALIST:UPDATE_FORBIDDEN
+* TABLEAU-SPECIALIST:UPDATE_FORBIDDEN_PERMISSION
+
+
+== Sources
+
+[[NewOrModifiedDatasourceListener]]
+== On New Or Modified Data Source
+`<tableau-specialist:new-or-modified-datasource-listener>`
+
+Initiates a flow when a data source is created or updated.
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Since a| String |  Date in the yyyy-MM-dd'T'HH:mm:ss'Z' format, for example, 2014-01-01T00:00:00Z. If this field is empty, this operation retrieves the selected objects from the time the Mule app is started. |  |
+| Config Ref a| ConfigurationProvider |  Name of the configuration used to execute this component. |  | x
+| Primary Node Only a| Boolean |  Determines whether to execute this source on only the primary node when running Mule instances in a cluster. |  |
+| Scheduling Strategy a| scheduling-strategy |  Configures the scheduler that triggers the polling. |  | x
+| Redelivery Policy a| <<RedeliveryPolicy>> |  Defines a policy for processing the redelivery of the same message. |  |
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Object
+| Attributes Type a| Any
+|===
+
+=== For Configurations
+
+* <<TableauSpecialistConfig>>
+
+
+
+== Types
+
+[[Reconnection]]
+=== Reconnection
+
+Configures a reconnection strategy for an operation.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Fails Deployment a| Boolean | What to do if, when an app is deployed, a connectivity test does not pass after exhausting the associated reconnection strategy:
+
+* `true`
```

Comment:
The formatting of this list and some of the ones later on are incorrect. I think they are missing the + signs.